packages/components/src/components/form/Checkbox/Checkbox.tsx

```diff
@@ -87,13 +89,19 @@ export const Checkbox = ({
 
     return (
         <Wrapper
+            data-test={dataTest}
```

Comment:
It already accepts `data-test` as a part of `rest`. It's passed [here](https://github.com/trezor/trezor-suite/blob/45f7b8bc1266df82223abfbee20f3f5b61e538eb/packages/suite/src/components/firmware/CheckSeedStep.tsx#L148) for example. :)

Reply:
Cool. I removed that 9da580e1edb8550f24d0d4bcedf138449ef1f283